### ISSUE TITLE
bootstrap not loading from layout.html

### ISSUE THREAD

**Issue by repcsi** [talkpython/modern-apis-with-fastapi#7]
Hello Mike,

It seems that the stylesheet link is no longer valid that you included in layout.html:
`<link href="//oss.maxcdn.com/libs/twitter-bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">`
I had to change it to:
`<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css">`

Refer to https://bootstrapdocs.com/v3.0.3/docs/getting-started/
and check "Bootstrap CDN"
I went ahead and used bootstrap beta 5, which looks way nicer btw, check it out if you want to (also supports integrity checks which is a plus):
`<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">`

**Comment by mikeckennedy** [comment]
Thank you @repcsi I just fixed it in the course. I personally hate CDNs for exactly this reason. But for courses, it's sometimes easier to just put that in there as a CDN resource rather than covering static content, etc right away.

As for BS5, yes, it's much nicer. But that theme was built with 3 and we don't really use it so I figured I'd just keep the old one in place - could be convinced either way. But I'd start a new project with 5 for sure. :)
